tools/clang/test/CodeGenHLSL/quick-test/local_res_array.hlsl

```diff
@@ -0,0 +1,17 @@
+// RUN: %dxc -T cs_6_0 -E main  -fcgl %s | FileCheck %s

```

Comment:
Need to add a test to check for the error message that says we cannot support dynamic indexing.

Also add test for error message for array dimension greater than one.